h/api/test/presenters_test.py

```diff
@@ -78,6 +114,52 @@ def test_link_generators_called_with_request_and_annotation(self):
 
         dummy_link_generator.assert_called_once_with(request, annotation)
 
+    def test_text(self):
+        request = DummyRequest()
+        ann = mock.Mock(text='It is magical!')
+        presenter = AnnotationBasePresenter(request, ann)
+
+        assert 'It is magical!' == presenter.text
```

Comment:
The earlier asserts in this diff use `assert actual <predicate> expected` but here there is a switch to `assert expected <predicate> actual`. I'd prefer it was consistent if possible.